packages/jest-cli/src/TestScheduler.js

```diff
@@ -351,7 +351,10 @@ export default class TestScheduler {
     aggregatedResults: AggregatedResult,
     watcher: TestWatcher,
   ): Promise<void> {
-    if (this._globalConfig.bail && aggregatedResults.numFailedTests !== 0) {
+    if (
+      this._globalConfig.bail !== 0 &&
+      aggregatedResults.numFailedTests >= this._globalConfig.bail
```

Comment:
this will still be true if `bail` is explicitly set to `false`. Shouldn't we normalize it just in case?

Reply:
There shouldn't be any case where `bail` gets to this point that isn't a number: https://github.com/facebook/jest/pull/7335/files#diff-7121e12d406ff9a3060a13ee808632de